packages/app-scripts-preact/babel.config.json

```diff
@@ -3,5 +3,5 @@
     "pragma": "h",
     "pragmaFrag": "Fragment"
   }], "@babel/preset-typescript"],
-  "plugins": ["@babel/plugin-syntax-import-meta", "babel-plugin-macros"]
+  "plugins": ["@babel/plugin-syntax-import-meta", "babel-plugin-macros", "react-refresh/babel"]
```

Comment:
nit: can you remove `babel-plugin-macros` from this list? we used to support this by default but it's now opt-in for each developer.